middleware/locale/method/cookie_test.go

```diff
@@ -0,0 +1,33 @@
+package method_test
```

Comment:
I think I would stick with Go's best practice to put tests in the dedicated `*_test` package. This prevents the use of unexported functions in the test.